packs/feats/divine-castigation.json

```diff
@@ -30,16 +30,61 @@
         },
         "rules": [
             {
-                "key": "EphemeralEffect",
+                "domain": "all",
+                "key": "RollOption",
+                "option": "divine-castigation",
+                "placement": "spellcasting",
+                "toggleable": true
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "divine-castigation",
+                    "self:trait:holy",
+                    {
+                        "or": [
+                            "item:slug:harm",
+                            "item:slug:heal"
+                        ]
+                    }
+                ],
+                "property": "traits",
+                "value": "holy"
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
                 "predicate": [
-                    "item:type:spell",
-                    "item:slug:heal",
-                    "target:trait:fiend"
+                    "divine-castigation",
+                    "self:trait:unholy",
+                    {
+                        "or": [
+                            "item:slug:harm",
+                            "item:slug:heal"
+                        ]
+                    }
                 ],
-                "selectors": [
-                    "damage-received"
+                "property": "traits",
+                "value": "unholy"
+            },
+            {
+                "key": "DamageDice",
```

Comment:
Not sure if there's a better way to automate this, given that it only become spirit and damages creatures with the opposing trait.

Reply:
This automates something other than what Divine Castigation does.

Reply:
I don't believe it's currently possible to modify the damage type on application. I'll remove this RE then?